website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -0,0 +1,176 @@
+
+## Overview
+
+<expandable alt_header="How does dbt Explorer help with data quality?" >
+
+dbt Explorer makes it easy and intuitive to understand your entire lineage &mdash; from data source to the reporting layer &mdash; so you can troubleshoot, improve, and optimize your pipelines. With built-in features like project recommendations and model performance analysis, you can be sure you have appropriate test and documentation coverage across your estate and quickly spot and remediate slow-running models. With column-level lineage, you can quickly identify the potential downstream impacts of table changes or work backwards to quickly understand the root cause of an incident. dbt Explorer gives teams the insights they need to improve data quality proactively, ensuring pipelines stay performant and data trust remains solid.
+
+</expandable>
+
+<expandable alt_header="How is dbt Explorer priced?" >
+
+With the current Preview, dbt Explorer is available to everyone on the dbt Cloud Enterprise and Team plans. Certain features within dbt Explorer, such as multi-project lineage and column-level lineage, are only available on the Enterprise plan. dbt Explorer can be accessed by users with developer and read-only seats during the Preview period.
+
+</expandable>
+
+<expandable alt_header="What happened to dbt Docs?" >
+
+dbt Explorer will become the default documentation experience for dbt Cloud customers. dbt Docs is still available but doesn't offer the same speed, metadata, or visibility as dbt Explorer and will become a legacy feaure.
+
+</expandable>
+
+## How dbt Explorer works
+
+<expandable alt_header="Can I use dbt Explorer on-prem or with my self-hosted dbt Core deployment?" >
+
+No. dbt Explorer is only available as a dbt Cloud user experience. dbt Explorer reflects the metadata from your dbt Cloud project(s) and their runs. 
+
+</expandable>
+
+<expandable alt_header="How does dbt Explorer support dbt Cloud environments?" >
+
+dbt Explorer defaults to the latest production state of a project. Support for staging and development (Cloud CLI and IDE) environments is coming soon. Users can only assign a single production and staging environment per dbt Cloud project. 
+
+</expandable>
+
+<expandable alt_header="How do I get started in Explorer? How does it update?" >
+
+Simply select **Explore** from the dbt Cloud top navigation bar. dbt Explorer automatically updates after each dbt Cloud run in the given project’s environment (production, by default). The dbt commands you run within the environment will generate and update the metadata in dbt Explorer, so make sure to run the correct combination of commands within the jobs of the environment; for more details, refer to [Generate metadata](/docs/collaborate/explore-projects#generate-metadata). 
```

Comment:
Since its going to move soon
```suggestion
Simply select **Explore** from the dbt Cloud navigation bar. dbt Explorer automatically updates after each dbt Cloud run in the given project’s environment (production, by default). The dbt commands you run within the environment will generate and update the metadata in dbt Explorer, so make sure to run the correct combination of commands within the jobs of the environment; for more details, refer to [Generate metadata](/docs/collaborate/explore-projects#generate-metadata). 
```